packages/offix-datastore/src/storage/LocalStorage.ts

```diff
@@ -116,6 +120,26 @@ export class LocalStorage {
     return this.adapter.query(storeName);
   }
 
+  /**
+   * Create store in underlying storage provider
+   *
+   * @param config
+   */
+  public addStore(config: IStoreConfig) {
```

Comment:
LocalStorage is the only one element that people interact with at the moment. We can add store and createStores - but I see those methods here only temporarily. Store creation is specific to the IndexedDB etc.

I would drop SQL lite here and see how we can create stores on demand (this will make it hard for us to track the versions of the stores etc so current approach of creating all stores upfront and init is the best. 

Same as replication creation on stores is final and once app is initialized we cannot recreate it etc. (which makes this API undocumented and hard to use)